cmd/gitserver/server/cleanup.go

```diff
@@ -1199,7 +1209,9 @@ func removeFileOlderThan(path string, maxAge time.Duration) error {
 		return nil
 	}
 
-	log15.Debug("removing stale lock file", "path", path, "age", age)
+	logger := log.Scoped("removeFileOlderThan", "removes path if its mtime is older than maxAge.")
+
+	logger.Debug("removing stale lock file", log.String("path", path), log.String("age", age.String()))
```

Comment:
I think you can use a duration here:

```suggestion
	logger.Debug("removing stale lock file", log.String("path", path), log.Duration("age", age))
```